.gitignore

```diff
@@ -19,3 +19,6 @@ Cargo.lock
 
 # Ignore the `crypto-framework` submodule
 crypto-framework/
+
+# Ignore the JS app I used to test the wasm API
+private-identity-audit/wasm/app
```

Comment:
Actually it might be a good idea to include the app. It could serve as a good example for others who want to use the wasm api. What do you think?

Reply:
I thought it would be a bit distracting to keep it. But we have a framework for testing the MERCAT work flow, this is similar to that. I'll look into the minimum structure I need to keep in order to build the app--I agree that it can be helpful when we do integration.